packages/storybook/stories/Stateful Client/UseSelector.stories.mdx

```diff
@@ -32,6 +31,42 @@ const ChatInfo = ({id: string, userName: string}) => {
 <ChatInfo {...chatInfoProps} />
 ```
 
+### Passing your own props
+
+When creating your own selector, you can pass in your own parameters into selector by using second parameter of selector function:
```

Comment:
When creating your own selector, you can inject your own custom properties through the second parameter of the selector function.